e2e/tests/keycloak-superset-flows.spec.ts

```diff
@@ -14,6 +14,34 @@ test.beforeEach(async ({ page }) => {
   superset = new Superset(page);
 });
 
+test('Logging out from Superset logs out the user from Keycloak.', async ({ page }) => {
```

Comment:
Probably, you should mention something like "_Logging out from Superset ends the session in Keycloak and logs out the user_"